index.html

```diff
@@ -11,21 +11,17 @@
     <link rel="stylesheet" href="http://yui.yahooapis.com/pure/0.2.1/pure-min.css">
     <link rel="stylesheet" href="css/fakeiteasy.css">
     <link rel="stylesheet" href="css/syntax.css">
-    <link href="http://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet" type="text/css">
+    <link href="http://fonts.googleapis.com/css?family=Ubuntu:400,700" rel="stylesheet" type="text/css">
     
   </head>
   <body>
     <div class="splash">
-      <div class="box splash-text">
-        <img class="floatright" src="img/talkingladies.png" alt="Two ladies talking about FakeItEasy">
-        <h1 class="splash-head">FakeItEasy</h1>
-        <h2 class="splash-subhead">The easy mocking library for .NET</h2>
-
-      </div>
+      <img src="img/fakeiteasy_logo_256.png" alt="FakeItEasy logo">
+      <h2>The easy mocking library for .NET</h2>
```

Comment:
We should probably be using a 512 image, sized to 256, to support high DPI scaled up to 200%. Or perhaps, we should think about switching to SVG. (Of course, you can't actually do any of this until I've uploaded the source images to our new repo.)